tests/UI/pages/BO/design/themeAndLogo/themeAndLogo/import.ts

```diff
@@ -26,15 +34,51 @@ class ImportTheme extends themeAndLogoBasePage {
     this.pageTitle = `Theme import • ${global.INSTALL.SHOP_NAME}`;
 
     this.importForm = 'form[name="import_theme"]';
+    // Form "Import from your computer"
+    this.zipFileUploadButton = '#import_theme_import_from_computer';
+    this.importFromYourComputerSubmit = `${this.importForm} div.row div:nth-child(1) div.card-footer button`;
+
     // Form "Import from the web"
     this.inputArchiveURL = '#import_theme_import_from_web';
-    this.importWebSubmit = `${this.importForm} > div.row > div:nth-child(2) div.card-footer button`;
+    this.importWebSubmit = `${this.importForm} div.row div:nth-child(2) div.card-footer button`;
+
+    // From FTP
+    this.selectArchive = '#import_theme_import_from_ftp';
+    this.importFTPSubmit = `${this.importForm} div.row div:nth-child(3) div.card-footer button`;
+  }
+
+  /**
+   * Import theme from your computer
+   * @param page {Page} Browser tab
+   * @param path {string} Theme path to import
+   * @returns {Promise<void>}
+   */
+  async importFromYourComputer(page: Page, path: string): Promise<void> {
+    await this.uploadFile(page, this.zipFileUploadButton, path);
+    await page.click(this.importFromYourComputerSubmit);
   }
 
+  /**
+   * Import theme from web
+   * @param page {Page} Browser tab
+   * @param themeUrl {string} Theme URL link to import
+   * @returns {Promise<void>}
+   */
   async importTheme(page: Page, themeUrl: string): Promise<void> {
```

Comment:
```suggestion
  async importFromWeb(page: Page, themeUrl: string): Promise<void> {
```

Do you think you can rename for keeping consistency ?

Reply:
Done :+1: